### ISSUE TITLE
Unexpected token: | 

### ISSUE THREAD

**Issue by ovcharenkovv** [phpmd/phpmd#516]
In PHP 7.1 was added the ability catching multiple exception types (https://wiki.php.net/rfc/multiple-catch).

Example code:

```php
<?php
try {
    // Some code...
} catch (ExceptionType1 | ExceptionType2 $e) {
    // Code to handle the exception
}    
```

Error:
`Unexpected token: |, line: x, col: y, file: z.`

**Comment by ravage84** [comment]
Which version of PHPMD and Pdepend have you used?

**Comment by ravage84** [comment]
This is probably due to missing support for PHP 7.1 in Pdepend.

**Comment by emirb** [comment]
Addressed in https://github.com/pdepend/pdepend/pull/349

**Comment by kylekatarnls** [comment]
Fixed in PDepend 2.5.1.

**Comment by ovcharenkovv** [comment]
Thank you mate.
